db/init.sql

```diff
@@ -4,13 +4,154 @@ CREATE DATABASE mm_db;
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
 CREATE TYPE user_role AS ENUM ('admin', 'teacher', 'student');
+CREATE TYPE attempt_state AS ENUM ('submitted', 'graded');
+-- NOTE(nrydanov): Need to think of other types together
+CREATE TYPE block_type AS ENUM ('task', 'markdown');
```

Comment:
What about `poll` and `quiz` types? Is it included in `markdown` part?

Reply:
We decided that it's too early to add those types. We can easily change it in future if we want.